screens/HomeScreen.js

```diff
@@ -1,35 +1,56 @@
 import React, { useState } from 'react';
-import { Image, Platform, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import { Button, Input } from 'react-native-elements';
+import { Button, Platform, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { Input } from 'react-native-elements';
 import _sumBy from 'lodash/sumBy';
-import { ScrollView } from 'react-native-gesture-handler';
+
 import * as WebBrowser from 'expo-web-browser';
 
 import { MonoText } from '../components/StyledText';
 import { TextList } from '../components';
 
+const ALL_EXPENSES = [
+  { id: 1, name: 'Buy a book', amount: 20 },
+  { id: 2, name: 'Buy dog food', amount: 5 },
+  { id: 3, name: 'Book a flight ticket', amount: 225 },
+  { id: 4, name: 'Buy speakers', amount: 300 },
+  { id: 5, name: 'Book a concert ticket', amount: 50 },
+  { id: 6, name: 'Buy birthday present', amount: 100 },
+  { id: 7, name: 'Buy boots', amount: 400 },
+  { id: 7, name: 'Buy boots', amount: 1000 },
```

Comment:
It is dummy data, right, but having the same "ID" is bad...IDs should be unique in any system.

Reply:
right! this is a typo that coms from testing , last element should not be there ..